editor/scripts/bundle.py

```diff
@@ -140,18 +140,16 @@ def rmtree(path):
     if os.path.exists(path):
         shutil.rmtree(path, onerror=remove_readonly_retry)
 
-def mac_certificate():
-    # This certificate must be installed on the computer performing the operation
-    certificate = 'Developer ID Application: Midasplayer Technology AB (ATT58V7T33)'
-    if exec_command(['security', 'find-identity', '-p', 'codesigning', '-v']).find(certificate) >= 0:
-        return certificate
+def mac_certificate(codesigning_identity):
+    if exec_command(['security', 'find-identity', '-p', 'codesigning', '-v']).find(codesigning_identity) >= 0:
+        return codesigning_identity
     else:
         return None
 
-def sign_files(bundle_dir):
-    certificate = mac_certificate()
+def sign_files(codesigning_identity, bundle_dir):
+    certificate = mac_certificate(codesigning_identity)
     if certificate == None:
-        print("Warning: Codesigning certificate not found, files will not be signed")
+        print("Warning: Codesigning certificate not found for signing identity %s, files will not be signed" % (codesigning_identity))
```

Comment:
Tbh, shouldn't this be an error, if the identity wasn't found?

Reply:
Yes, maybe. The current setup is that if a signing identity isn't explicitly provided we use the Defold Foundation one as default value. This would mean that other developers who doesn't have access to the Defold Foundation identity must provide their own.

Or we could remove the default value and skip code signing if codesigning_identity is None. Or we could use the --skip-codesign flag also for the editor build.